src/survey-renderer/elements/elements.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
```

Comment:
I haven't put up style guides yet, but 4 spaces rather than tabs please. They render inconsistently if you don't use spaces.